packages/components/src/components/carousel/carousel.examples.md

```diff
@@ -206,6 +221,12 @@ export default class Code extends Vue {
   ${this.getSlides(3)}
 </p-carousel>`;
 }
+  alignHeaderModel = 'left';
+  get alignHeader() {
+    return `<p-carousel align-header="${this.alignHeaderModel}" heading="${this.basicHeading}" description="${this.basicDescription}">
```

Comment:
Indentation is messed up in the final result:
<img width="630" alt="Bildschirm­foto 2023-02-06 um 10 30 21" src="https://user-images.githubusercontent.com/4380295/216939378-a5766262-007b-491a-9a6a-fd686789f062.png">

Reply:
Fixed it